playground/app.js

```diff
@@ -352,6 +352,14 @@ class App extends Component {
         <div className="col-sm-5">
           {!this.state.form ? null :
             <Form
+              id="test-id"
+              className="testclass big"
+              name="test-name"
+              method="put"
+              target="_blank"
+              autocomplete="off"
+              enctype="multipart/form-data"
+              acceptcharset="ISO-8859-1"
```

Comment:
Not sure these should be in the playground. On the other hand I think these attributes should be documented :)

Would you mind adding a section to the docs?